src/sonic-config-engine/tests/sample_output/py2/l2switch.json

```diff
@@ -4,162 +4,194 @@
         "Ethernet8": {
             "alias": "fortyGigE0/8",
             "lanes": "37,38,39,40",
-            "admin_status": "up"
+            "admin_status": "up",
+            "speed": "10000"
         },
         "Ethernet0": {
             "alias": "fortyGigE0/0",
             "lanes": "29,30,31,32",
-            "admin_status": "up"
+            "admin_status": "up",
+            "speed": "10000"
         },
         "Ethernet4": {
             "alias": "fortyGigE0/4",
             "lanes": "25,26,27,28",
-            "admin_status": "up"
+            "admin_status": "up",
+            "speed": "10000"
```

Comment:
>speed [](http://example.com/codeflow?start=13&length=5)

Top question: in concept, there should be some default port speed. You may check swss implementation. Maybe we need to fix src/sonic-yang-models/yang-models/sonic-port.yang?

Reply:
Missing port speed in config_db is by design. ref: [https://github.com/Azure/SONiC/wiki/L2-Switch-mode](https://github.com/Azure/SONiC/wiki/L2-Switch-mode)

Reply:
then port speed is not mandatory, I will update yang models instead.

Reply:
Sorry maybe I am wrong. I do see port speed in l2switch config. Need to double check.